k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -5,11 +5,38 @@ DIR=$(dirname "$SCRIPT")
 
 function parseSolution()
 {
-  echo "$($DIR/parse_scripts/parse_yaml.sh $DIR/$solution_yaml $1)"
+  echo "$($DIR/parse_scripts/parse_yaml.sh $solution_yaml $1)"
+}
+
+function usage() {
+  echo -e "\n** Script to Collect CORTX Cloud Logs for Support Bundle **\n"
```

Comment:
> echo -e "\n** Script to Collect CORTX Cloud Logs for Support Bundle **\n"

change to
echo -e "\n** Generate CORTX Cluster Support Bundle **\n"

Reply:
changed as per the suggestion 👍